plugin.image.dumpert/addon.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import os
+import sys
+import urllib.parse
+import xbmc
+import xbmcaddon
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.image.dumpert").getAddonInfo('path'), 'resources', 'lib'))
+sys.path.append(LIB_DIR)
```

Comment:
no need to use sys.path.append inside the same addon. Why are you doing this instead of just importing the modules you need?

Reply:
How will the addon otherwise know it has to import a module from here: resource/lib?

Reply:
Assuming you have a something.py file in resources lib you simply need `from resources.lib import something`

In python2 you need a `__init__.py` file in resources/lib folder

Reply:
Fixed.